app/src/main/java/com/simplecity/amp_library/ui/screens/playlist/detail/PlaylistDetailFragment.kt

```diff
@@ -149,6 +150,8 @@ class PlaylistDetailFragment :
 
         adapter = ViewModelAdapter()
 
+        ShuttleApplication.adapter = adapter
```

Comment:
So the adapter is instantiated in the `ShuttleApplication` when the app starts, but then replaced by this one instantiated in the Fragment? That doesn't seem right. Could you explain why this is happening in a code comment?